ui/src/components/entity/citation/PaperSummary.tsx

```diff
@@ -168,12 +154,52 @@ export default class PaperSummary extends React.PureComponent<Props, State> {
                   </div>
                 </Tooltip>
               ) : null}
+              {!!paper.inboundCitations ? (
+                <Tooltip
+                  placement="bottom-start"
+                  title={
+                    <React.Fragment>
+                      <strong>
+                        {paper.inboundCitations} citation
+                        {paper.inboundCitations !== 1 ? "s" : ""}
+                      </strong>
+                    </React.Fragment>
+                  }
+                >
+                  <div className="paper-summary__metrics__metric">
+                    <InboundCitationIcon width="12" height="12" />
+                    {paper.inboundCitations}
+                  </div>
+                </Tooltip>
+              ) : null}
+              {!!paper.outboundCitations ? (
+                <Tooltip
+                  placement="bottom-start"
+                  title={
+                    <React.Fragment>
+                      <strong>
+                        {paper.outboundCitations} reference
+                        {paper.outboundCitations !== 1 ? "s" : ""}
```

Comment:
I feel like there is an issue with the whitespace output here...but I can't put my finger on it.

Reply:
There doesn't seem to be one in practice. I know there are libraries like `pluralize` that handle this _and_ be locale aware, maybe that can be a future thing.